src/services/VersionCheckerService.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * (C) Symbol Contributors 2021 (https://nem.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License.
+ *
+ */
+import { appConfig } from '@/config';
+import fetch from 'node-fetch';
+
+export type LatestVersionObject = {
+    versionTag: string;
+    downloadUrl: string;
+};
+
+export class VersionCheckerService {
+    /**
+     * Get latest version tag object
+     * @return {Promise<{versionTag:string, downloadUrl:string}>}
+     */
+
+    public async getLatestVersionTag(): Promise<LatestVersionObject> {
```

Comment:
should we rename this to `getNewerVersionTag` ? 

Current name implies that it returns the latest version no matter what. But it returns undefined if the current version is the latest. Should update the method description as well.

Reply:
IMO this method is kind of redundant. `requestGithubRepo` returns the same Object.
We can just use that.